src/libraries/System.Console/tests/CancelKeyPress.Unix.cs

```diff
@@ -78,7 +78,7 @@ public void ExitDetectionNotBlockedByHandler()
 
             // Release CancelKeyPress
             mre.Set();
-        }).Dispose();
+        }, new RemoteInvokeOptions() { CheckExitCode = false }).Dispose();
```

Comment:
Should we instead be checking for a specific exit code... I'm guessing it was 130?